Packs/Use_Case_Builder/Scripts/GettingStartedWithXSOAR/GettingStartedWithXSOAR.py

```diff
@@ -0,0 +1,34 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+# disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+WELCOME_MESSAGE = '''\n
+# Welcome to the XSOAR Use Case Builder!
+
+**Please Watch this video to learn the basics of XSOAR.**
+'''
+
+
+def download_clip(verify_ssl=False):
+    res = requests.get(
+        'https://github.com/joe-cosgrove/xsoar-videos/blob/main/Cortex%20XSOAR%20In%20Under%205%20Minutes.mp4?raw=true',
+        verify=verify_ssl
+    )
+    res.raise_for_status()
+
+    return res.content
+
+
+def main():
+    data = download_clip()
+    entry = fileResult('Getting Started with XSOAR', data)
+    entry.update({
+        "Type": 10,  # EntryVideoFile
```

Comment:
Please use InfoFile entry type as described [here](https://xsoar.pan.dev/docs/integrations/yaml-file#entry-types)

Reply:
Type 10 will actually render the video to be played. Is there a different infoFile type i can use that will maintain the functionality?

Reply:
My bad, you can keep it